Model/Cron.php

```diff
@@ -1891,48 +1891,59 @@ protected function _getFraudManualReviewAcceptStatus()
     }
 
     /**
+     * Get all the entries in the adyen_order_payment and compare their total to the order grand total
+     *
      * @param int $paymentId
      * @param string $orderCurrencyCode
      * @return bool
      */
-    protected function _isTotalAmount($paymentId, $orderCurrencyCode)
+    protected function _isTotalAmount($paymentId, $orderCurrencyCode, $captured = false)
     {
+        $orderId = $this->_order->getId();
         $this->_adyenLogger->addAdyenNotificationCronjob(
-            'Validate if AUTHORISATION notification has the total amount of the order'
+            'Validate if after processing current AUTHORISATION notification, the full amount has been authorised'
         );
 
-        // get total amount of the order
-        $grandTotal = (int)$this->_adyenHelper->formatAmount(
+        // Get total amount of the order
+        $grandTotal = $this->_adyenHelper->formatAmount(
             $this->orderAmount,
             $this->orderCurrency
         );
 
-        // check if total amount of the order is authorised
+        // Get total amount currently authorised
         $res = $this->_adyenOrderPaymentCollectionFactory
             ->create()
-            ->getTotalAmount($paymentId);
+            ->getTotalAmount($paymentId, $captured);
 
         if ($res && isset($res[0]) && is_array($res[0])) {
             $amount = $res[0]['total_amount'];
             $formattedOrderAmount = $this->_adyenHelper->formatAmount($amount, $orderCurrencyCode);
             $this->_adyenLogger->addAdyenNotificationCronjob(
                 sprintf(
-                    'The grandtotal amount is %s and the total order amount that is authorised is: %s',
+                    'The grand total amount for order %s is %s and the currently authorised order amount is: %s',
+                    $orderId,
```

Comment:
I see $orderId is the DB ID, but the merchantReference/IncrementId would be more useful in this log message.

Reply:
070ce8e